### DISCUSSION TITLE
How to add suggestions to an QInput field

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#10797] [General - Components / Directives / etc] [answered]
I use a QField to input tags:

```
     <q-input model-value='undefined' v-model='note.newTag' @change='addTag(note)' label="tags" dense clearable
               clear-icon='close' borderless>
        <template v-slot:before>
          <q-icon name='add_circle_outline'/>
        </template>
      </q-input>

```
Is it possible to add a suggestion dropdown from which I could choose some proposals? (a `computed` list)

**Comment by pdanpdan** [comment, marked as answer]
Sure. Already available as example in QSelect

**Reply by wsw70** [reply]
Thank you. using Qselect drastically simplified my code as both the input part, the chips and the filtering is right there.
